qpc/request.py

```diff
@@ -167,7 +167,12 @@ def request(method, path, params=None, payload=None,
             log.error('Unsupported request method %s', method)
             parser.print_help()
             sys.exit(1)
-        log_request_info(method, log_command, url, payload, result.status_code)
+        try:
+            log_request_info(method, log_command,
+                             url, result.json(), result.status_code)
+        except ValueError:
+            log_request_info(method, log_command,
```

Comment:
Could try and capture the response text.  Instead of None

Reply:
Done 👍